crates/solver/src/interactions/allowances.rs

```diff
@@ -440,24 +426,4 @@ mod tests {
             },
         );
     }
-
-    #[tokio::test]
-    async fn fetch_fails_on_batch_errors() {
```

Comment:
Since our web3 instance auto-batches under the hood the new code is no longer aware of any special cases related to batching so this test can be dropped.